README.md

```diff
@@ -1 +1,115 @@
-# leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
- [ ] It would be nice if you fix your FAQ link because right now it is broken 😨 Kindly fix it or remove it (Since you don't have any dedicated section for this link in the whole readme file) to make your readme descriptive and professional 👍